Samples/src/SimpleGrover/SimpleGrover.qs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+namespace Microsoft.Quantum.Samples.SimpleGrover {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Arithmetic;
+
+    // This operation adds a (-1)-phase to the marked state(s)
+    operation ReflectMarked (inputQubits : Qubit[]) : Unit is Adj + Ctl {
```

Comment:
You never call the adjoint or a controlled version of the function, so `Adj + Ctl` could be removed.